### PR TITLE
School Update

### PR DESCRIPTION
**Pull request by Nikademus**
Closes #

## Changes proposed:
- School format and GUI update. 

Merge please :D

### CODE REVIEW

File: pages/src/source/buildings/school.md [hunk 1]
```diff
@@ -6,9 +6,8 @@ layout: default
 
 {% capture content %}
 ### Note: The School cannot be built until you finish the research in the [University](../../source/buildings/university).
-<br>
 
-The School is where the Teacher will level up the Pupils' (children) skills. Paper will increase the leveling speed of the Pupils. 
+The School is where the Teacher will level up the Pupils' (children) Intelligence skill. Paper will increase the leveling speed of the Pupils.
```
Comment (Thodor12) [inline pages/src/source/buildings/school.md:10]: "School" is not really a special name in the regard that it should be capitalized, please make it lowercase "school", same with "Teacher" and "Pupils'"

**Review by Thodor12 — CHANGES_REQUESTED** [review]

File: pages/src/source/buildings/school.md (hunk 1 above)
Comment (Nikademus) [reply pages/src/source/buildings/school.md:10]: I thought the hut and worker names were capitalized intentionally, so I've been keeping them that way.

Comment (Talyda) [reply pages/src/source/buildings/school.md:10]: I thought they were too TBH - made sense to me that the workers and hut names were capitalised

Comment (anonymous563) [reply pages/src/source/buildings/school.md:10]: Workers and hut names are capitalized intentionally.

**Review by anonymous563 — APPROVED** [review]

**Pull request merged**